is/object_of_test.ts

```diff
@@ -98,4 +98,46 @@ Deno.test("isObjectOf<T>", async (t) => {
       >(true);
     }
   });
+
+  await t.step("if 'predObj' has prototype properties", async (t) => {
+    const prototypeObj = {
+      a: is.Number,
+      b: is.Boolean,
+    };
+    // deno-lint-ignore ban-types
+    const predObj2 = Object.assign(Object.create(prototypeObj) as {}, {
+      c: is.String,
+    });
+
+    await t.step("returns true on T object that omits prototype", () => {
+      assertEquals(isObjectOf(predObj2)({ c: "a" }), true);
+      assertEquals(
+        isObjectOf(predObj2)({ c: "a", d: "ignored" }),
+        true,
+      );
+      assertEquals(
+        isObjectOf(predObj2)(Object.assign(() => void 0, { c: "a" })),
+        true,
+      );
```

Comment:
I'd like to see the result of `{a: "a", b: 0, c: "a"}`. I guess it is `true`, right?